examples/TestDriver/src/pages/basics.js

```diff
@@ -10,6 +10,7 @@ import {
   TouchableWithoutFeedback,
   View,
 } from 'react-native';
+import { Switch } from 'native-base';
```

Comment:
Cool, but let's make sure we try the built-in RN switch as well, please!

Reply:
Done